docs/reference/snapshot-restore/register-repository.asciidoc

```diff
@@ -127,9 +130,27 @@ Other repository types are available through official plugins:
 
 * {plugins}/repository-hdfs.html[Hadoop Distributed File System (HDFS)]
 
-You can also use alternative implementations of these repository types, such as
-MinIO, as long as they're compatible. To verify a repository's compatibility,
-see <<snapshots-repository-verification>>.
+You can also use alternative storage implementations with these repository
+types, as long as the alternative implementation is fully compatible. For
+instance, https://minio.io[MinIO] provides an alternative implementation of the
```

Comment:
"a compatible alternative implementation"? Otherwise the "so" later does not refer to compatibility, just some implementation.

Reply:
I see what you mean. Instead, I changed the `so` for an `and` to make the sentence more neutral about the level of compatibility offered by Minio.